__mocks__/request-promise.js

```diff
@@ -0,0 +1,47 @@
+import {getConfig} from '../src/__mocks__/helpers';
+
+
+function request(url) {
+  // TODO get rid of those globals
+  switch (global.MOCK_REQUEST_SCENARIO) {
```

Comment:
I know. the point is that the function call is from the tested function. So i cannot pass it to this function.